core/controllers/base_test.py

```diff
@@ -269,6 +271,31 @@ def test_i18n_keys(self):
                     self.log_line('- %s' % key)
                 self.log_line('')
 
+    def alphabetic_test_i18n_keys(self):
+        """Tests that all JSON files are arranged in aplhabetical order."""
+        filenames = os.listdir(
+            os.path.join(os.getcwd(), self.get_static_asset_filepath(),
+                         'assets', 'i18n'))
+        for filename in filenames:
+            json_file_content = json.loads(utils.get_file_contents(
```

Comment:
Hello @wxyxinyu . Thanks for your comments on this pull request. I did it this way because when trying to check that each line follows the previous one lexicographically, I run into a problem of unicode characters when using the sorted function and comparing to the original file. I've tried to do it the way you are doing without using the sorted function but nothing seems work. That is why I decided to go this way. So my problem is even though I thought of what you are proposing, I was not able to do it that way.